layouts/partials/head.html

```diff
@@ -89,10 +85,12 @@
 <meta property="og:type" content="{{ $ogType }}">
 
 {{ if .HasShortcode "mermaid" }}
-<!-- Copied from https://unpkg.com/mermaid@8.11.2/dist/mermaid.min.js -->
+<!-- Copied from https://unpkg.com/mermaid@8.13.4/dist/mermaid.min.js -->
 <script async src="{{ "js/mermaid.min.js" | relURL }}"></script>
```

Comment:
I think it's better to move the file and include the version in the filename. It would be frustrating if we had to deal with bugs caused by caching problems.

```suggestion
<!-- Copied from https://unpkg.com/mermaid@8.13.4/dist/mermaid.min.js -->
<script async src="{{ "js/mermaid-8.13.4.min.js" | relURL }}"></script>
```

Reply:
Very good idea but will save for future PR. I want to avoid changes that do not impact functionality at this point.